src/app/shared/picker/color-picker/color-picker.component.ts

```diff
@@ -100,13 +105,20 @@ export class ColorPickerComponent implements OnInit, AfterViewInit {
       chooseText: this.i18n({id: 'button.choose', value: 'Choose'}),
       preferredFormat: 'hex',
       containerClassName: 'spectrum-container',
-      clickoutFiresChange: true,
+      clickoutFiresChange: false,
       change: function(color) {
+        __this.savingColor = true;
         __this.select(color.toHexString());
       },
       move: function(color) {
         __this.preview(color.toHexString());
       },
+      hide: function(color) {
+        if (!__this.savingColor) {
+          __this.colorChange.emit(__this.originalColor);
+        }
+        __this.savingColor = false;
```

Comment:
I'm not sure if i inderstand correctly, but `__this` refers to component's `this`?

Reply:
Yes, this is an old construct threre. I did not change it. Should I? I could probably use bind...

Reply:
Thanks! Now it's more clear what's going on!